tests/tests/test_compressor.py

```diff
@@ -240,7 +240,12 @@ def test_csshtmljsminify(self):
     def test_uglifyjs(self):
         self._test_compressor('pipeline.compressors.uglifyjs.UglifyJSCompressor',
             'js', 'pipeline/compressors/uglifyjs.js')
-        
+
+    @skipUnless(settings.HAS_NODE, "requires node")
```

Comment:
@mathjazz This will need an update to the `package.json` so terser is actually installed for the tests.

Reply:
I thought https://github.com/jazzband/django-pipeline/pull/696/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519 does that?

Reply:
You're right, I'm blind :)

Reply:
![](https://media.giphy.com/media/v9rfTQBNqdsSA/giphy.gif)